### ISSUE TITLE
ropm clean does not appear to work

### ISSUE THREAD

**Issue by georgejecook** [rokucommunity/ropm#5]
- did ropm install rooibos
- package had wrong folder name (my problem, not ropm)
- fixed my package
- did ropm clean
- folders were all still there, unchanged

**Comment by TwitchBronBron** [comment]
strange. The ropm clean function will walk through the host project's `rootDir` and find and delete every roku_modules folder, and then delete any empty parent folders of roku_modules folders. I know that I tested this locally at one point. Did you perhaps change the rootDir of your host project? 

I don't have a `ropm clean` spec just yet, but internal to the `ropm install` command it actually leverages the `ropm clean` command. This test proves that files get cleaned up: 
https://github.com/rokucommunity/ropm/blob/master/src/commands/InstallCommand.spec.ts#L160

**Comment by georgejecook** [comment]
easy to isolate

```
mkdir r
cd r
mkdir src
mkdir src/source
mkdir src/components
ropm init
#update package.json to add
  "ropm": {
    "rootDir": "src"
  },
ropm install rooibos-roku
ls src/source/roku_modules/rooibosroku
ropm clean
ls src/source/roku_modules/rooibosroku
```

**Comment by TwitchBronBron** [comment]
Fixed by #8.
